Description: Upstream changes introduced in version 1.38-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 fio (1.38-1) unstable; urgency=low
 .
   * New Upstream Version:
     * Should build on ARMv7 now, thanks Jamie Bennett (Closes: #573027).
 .
 The person named in the Author field signed this changelog entry.
Author: Martin Steigerwald <ms@teamix.de>
Bug-Debian: http://bugs.debian.org/573027

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- fio-1.38.orig/Makefile
+++ fio-1.38/Makefile
@@ -42,9 +42,9 @@ endif
 endif
 
 INSTALL = install
-prefix = /usr/local
+prefix = /usr
 bindir = $(prefix)/bin
-mandir = $(prefix)/man
+mandir = $(prefix)/share/man
 
 %.o: %.c
 	$(QUIET_CC)$(CC) -o $*.o -c $(CFLAGS) $<
